src/utils/data-helper.js

```diff
@@ -2,10 +2,26 @@ import React from 'react'
 
 import moment from 'moment'
 
-export const formatData = data =>
-  moment(data).isValid()
-    ? moment(data).format('llll')
-    : data
+export const formatData = (data, column) => {
+  if(isOfDateType(column)) {
+    return moment(data).isValid()
+      ? moment(data).format('llll')
+      : data
+  } else {
+    return data
+  }
+}
+
+export const isOfDateType = attribute => {
+  switch (attribute) {
+  case 'createdAt':
+  case 'updateAt':
+  case 'lockedAt':
+    return true
+  }
+  return false
+}
```

Comment:
`['createdAt', 'updatedAt', 'lockedAt'].includes(attribute)`

Reply:
Copy that ! Je connaissais pas la méthode merci ;)